src/lens.jl

```diff
@@ -141,31 +150,37 @@ compose(l::Lens) = l
 compose(::IdentityLens, ::IdentityLens) = IdentityLens()
 compose(::IdentityLens, l::Lens) = l
 compose(l::Lens, ::IdentityLens) = l
-compose(l1::Lens, l2 ::Lens) = ComposedLens(l1, l2)
-function compose(ls::Lens...)
+compose(l1::Lens, l2 ::Lens) = ComposedLens(l2, l1)
```

Comment:
I didn't touch the order of `ComposedLens`. Should I flip it as well?

Reply:
Yes it is confusing if they are reversed. Also its probably better to rename the fields of `ComposedLens` into `outer,inner`. But can be a seperate PR.